lib/rules/web/ocm_console/cluster_detail_addons_tab.xyaml

```diff
@@ -63,9 +67,12 @@ check_console_url:
            xpath: //div[text()='<addon_name>']/../div/a/button[text()='View in console']
 
 check_contact_support:
-    element:
-        selector:
-            xpath: //div[text()='<addon_name>']/..//a[@href='https://access.redhat.com/support/cases/#/case/new' and text()='Contact support']
+    elements:
+        - selector:
+            xpath: //div[text()='<addon_name>']
```

Comment:
As so many place need to click add-on tab first, you'd better to make click add-on name as am action and then write it like below in other action calls.
action: click_addon_card